### DISCUSSION TITLE
Handshake ok (few bytes traffic) but no internet surf from the client

### DISCUSSION THREAD

**Discussion by mfricano2021** [angristan/wireguard-install#210] [Help]
Hi,

I installed WG using wireguard-install.sh . I can connect from the client because I can see the handshake with wg show, i can see the download/upload counters moves a little.

But Internet goes in timeout when I try to surf from the client (allowed IPs are already in 0.0.0.0/0). I see in postup and predown some iptables alredy inserted by the script. Should I be able to use the VPS to reach internet or the base iptables must be modified?

Reading others issues can be also a MTU problem. I will try this afternoon. Any others suggestions?

Many thanks

**Comment by Minc3** [comment]
Hello mfricano2021,
It sounds like you need to edit /etc/sysctl.conf and uncomment #net.ipv4.ip_forward=1

sudo nano /etc/sysctl.conf
Remove # from #net.ipv4.ip_forward=1
Optional: ipv6: #net.ipv6.conf.all.forwarding=1
Then restart your system.

Let me know if this helped.

**Comment by ghazi98** [comment]
Hi there, i am going through the same issue. Handshake works but no internet access. I have already forwarded ports. Here is a link of my issue.

https://www.reddit.com/r/WireGuard/comments/lrpp7o/wireguard_not_working_with_mulitple_clients/gon5fd4/?context=3
